samples/test-resources/multiproject/app1/build.gradle

```diff
@@ -0,0 +1,42 @@
+plugins {
+    id("io.micronaut.minimal.application")
+    id("io.micronaut.test-resources-consumer")
+}
+
+version = "0.1"
+group = "demo"
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    annotationProcessor("io.micronaut:micronaut-http-validation")
+    implementation("io.micronaut:micronaut-http-client")
+    implementation("io.micronaut:micronaut-jackson-databind")
+    implementation("io.micronaut:micronaut-runtime")
+    implementation("jakarta.annotation:jakarta.annotation-api")
+    runtimeOnly("ch.qos.logback:logback-classic")
+    implementation("io.micronaut:micronaut-validation")
+}
+
+application {
+    mainClass.set("demo.Application")
+}
+
+micronaut {
+    runtime("netty")
+    testRuntime("junit5")
+    processing {
+        incremental(true)
+        annotations("demo.*")
+    }
+    testResources.targetProject = ':testresources'
```

Comment:
Frankly speaking, I never used Micronaut before, and everything in the build file looks understandable except `testResources.targetProject = ':testresources'`

It looks like there are three entities "testResources", "targetProject", and ":testresources", and none of them makes sense to me at this point 🤷‍♂️

Reply:
Have you considered regular `dependencies` block instead?

For instance:

```groovy
dependencies {
  testResources(project(":testresources"))
}
```

I might be wrong, however, it does clarify the intention: "we depend on test resources provided by `:testresources` project). It would be compatible with project accessors.

Reply:
I have a doco PR coming which should clarify this.